docker/Makefile

```diff
@@ -20,11 +20,13 @@ P := grpc http kafka
 
 TARGETS := py3.6 py3.7 py3.8 py3.9 py3.10
 
+AGENT_VERSION := 0.7
```

Comment:
Hi. This is set  before running the make targets. Did you read this about how to build docker images? https://github.com/apache/skywalking-python/blob/master/docs/en/contribution/How-to-release-docker.md#how-to-build